CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+# PR [#30](https://github.com/theplant/appkit/pull/30)
+
+## Breaking Changes
+
+* `monitoring.NewInfluxdbMonitor` now returns a "close" `func()` to
+  allow termination of InfluxDB buffer introduced in #25.
```

Comment:
cc @bodhi and @lifenod 

Please note that once this PR got merged the Go apps that used this need to update this.

https://github.com/search?p=1&q=org%3Atheplant+NewInfluxdbMonitor&type=Code

Reply:
Reported in https://github.com/theplant/internal/issues/906.